lib/service/servicecfg/config.go

```diff
@@ -762,3 +769,11 @@ func verifyEnabledService(cfg *Config) error {
 	return trace.BadParameter(
 		"config: enable at least one of auth_service, ssh_service, proxy_service, app_service, database_service, kubernetes_service, windows_desktop_service, discovery_service, okta_service or jamf_service")
 }
+
+// SetLogLevel changes the loggers log level.
+func (c *Config) SetLogLevel(level slog.Level) {
```

Comment:
Ideally, the loggers on the process config should be the same as default loggers (for both `slog` and `logrus`). This is true if `ApplyFileConfig` is called before `SetLogLevel` because `applyLogConfig` [modifies `logrus` default logger](https://github.com/gravitational/teleport/blob/c6e63c8500f9d3b830e47e08329610f334804f24/lib/config/configuration.go#L686), [sets a new default logger for `slog`](https://github.com/gravitational/teleport/blob/c6e63c8500f9d3b830e47e08329610f334804f24/lib/config/configuration.go#L785), and sets both of them to the config.

This flow is used by the `teleport start` and `tctl` commands, which guarantees the other services will reuse the same base logger. I've tested with the DB service, App service, and Kube service, and they all were affected by the log-level changes.